storm-core/src/clj/backtype/storm/stats.clj

```diff
@@ -373,4 +360,1248 @@
     (ExecutorStats. (window-set-converter (:emitted stats) str)
       (window-set-converter (:transferred stats) str)
       specific-stats
-      rate)))
\ No newline at end of file
+      rate)))
+
+(defn- agg-bolt-lat-and-count
+  "Aggregates number executed and process & execute latencies across all
```

Comment:
OK I understand it is order of operations like thing.  I'm not sure a good way to explain it without being ambiguous.